Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.8.0-slim
+
+WORKDIR /python-support
+
+RUN apt-get update && apt-get install -y curl
```

`apt-get` be told to install only recommended, but not suggested packages. With `--no-install-recommends`, only the main dependencies are installed (smaller image).

```suggestion
RUN apt-get update && \
    apt-get -y upgrade && \
    apt-get install -y curl --no-install-recommends
```